src/sentry/workflow_engine/endpoints/organization_detector_types.py

```diff
@@ -43,7 +43,7 @@ def get(self, request, organization):
         type_slugs = [
             gt.slug
             for gt in grouptype.registry.get_visible(organization)
-            if gt.detector_handler is not None
+            if gt.detector_config is not None and gt.detector_config.handler is not None
```

Comment:
I think that eventually we can remove these double checks and require that all configs have a handler